public/scss/home.scss

```diff
@@ -0,0 +1,238 @@
+/* 88888888888888888    slider   8888888888888888888888 */
+.home {
+  .slider {
+    background-color: #fff;
+    margin-bottom: 20px;
+    padding: 15px 0 6px;
+    .slider_home {
+      position: relative;
+      margin: auto;
+      .prev,
+      .next {
+        cursor: pointer;
+        position: absolute;
+        top: 50%;
+        width: auto;
+        padding: 12px 16px;
+        margin-top: -22px;
+        color: white;
+        font-weight: 400;
+        font-size: 16px;
+        -webkit-transition: 0.6s ease;
+        -o-transition: 0.6s ease;
+        transition: 0.6s ease;
+        -webkit-user-select: none;
+        -moz-user-select: none;
```

Comment:
Why are you adding vendor prefixes manually.

Reply:
Done